packages/components/src/components/select/select.stories.ts

```diff
@@ -71,600 +61,313 @@ export default {
-        'form-control-input',
-        'form-control-help-text',
-        'combobox',
-        'display-input',
-        'listbox',
-        'tags',
-        'tag',
-        'tag__base',
-        'tag__content',
-        'tag__removable-indicator',
-        'clear-button',
-        'expand-icon'
-      ]
-    }
-  },
-  render: (args: { 'open-attr'?: string }) => {
-    delete args['open-attr'];
-
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'useTags' },
-        y: {
-          type: 'template',
-          name: 'sd-select::part(...){outline: solid 2px red}',
-          values: partsArr.map(part => {
-            return {
-              title: part,
-              value: `<style>#part-${part} sd-select::part(${part}){outline: solid 2px red; outline-offset: 2px} ::part(popup__content){overflow-y: visible} ::part(tag__removable-indicator){display: block} ::part(tag__content){display: block}</style><div id="part-${part}">%TEMPLATE%</div>`
-            };
-          })
-        }
-      },
-      constants: [
-        { type: 'attribute', name: 'useTags', value: true },
-        { type: 'attribute', name: 'value', value: 'option-1 option-2' },
-        clearableConstant,
-        helpTextConstant,
-        labelConstant,
-        multipleConstant
-      ],
-      args
-    });
-  }
+export const Disabled = {
+  render: () => html`
+    <div class="w-[400px]">
+      <sd-select size="lg" label="Label" placeholder="Disabled" placement="bottom" value="" disabled>
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+        <sd-option value="option-4">Option 4</sd-option>
+        <sd-option value="option-5">Option 5</sd-option>
+      </sd-select>
+    </div>
+  `
 };
 
 /**
- * Per default the select will indicate an error state when the input is invalid. Use the `style-on-valid` attribute to indicate a valid state as well.
+ * Use the `help-text` attribute to provide additional context or instructions.
  */
 
-export const StyleOnValid = {
-  parameters: {
-    controls: {
-      exclude: ['open-attr']
-    }
-  },
-  render: (args: { 'open-attr'?: string }) => {
-    delete args['open-attr'];
-
-    return html`<div class="h-[340px]">
-      ${generateTemplate({
-        options: {
-          classes: 'w-full'
-        },
-        axis: {
-          x: {
-            type: 'attribute',
-            name: 'style-on-valid'
-          }
-        },
-        constants: [fiveOptionsConstant, { type: 'attribute', name: 'value', value: '' }],
-        args
-      })}
-    </div>`;
-  },
-
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    await Promise.all([customElements.whenDefined('sd-select'), customElements.whenDefined('sd-option')]).then(
-      async () => {
-        const els = canvasElement.querySelectorAll('sd-select');
+export const HelpText = {
```

Comment:
Pls align it with what’s happening in sd-input right now
https://solid-design-system.github.io/solid/feature_sd-input-docs/?path=/docs/components-sd-input--docs#help%20text
<img width="1059" alt="Bildschirmfoto 2024-09-11 um 13 54 03" src="https://github.com/user-attachments/assets/d8e50ceb-690d-4adb-9449-386899266b3e">

Reply:
Updated